### DISCUSSION TITLE
Excited about this project!

### DISCUSSION THREAD

**Discussion by jaidetree** [jank-lang/jank#15] [Issue Triage] [answered]
Haven't dived in yet but really love the concept and look forward to seeing it develop. Not crazy about the name but I appreciate the ambition either way! Hope you don't mind but wanted to show some support.

**Comment by jeaye** [comment]
Thanks for reaching out, Jay! Also, thank you for that one time sponsorship. :)

I, too, am really excited about what's upcoming for jank; hopefully the name grows on you.

**Comment by rawsp33d** [comment]
Like the user above, I'm quite excited to see where this project goes, and if it goes where I think a lot of us think it can, it might end up being a new favorite language for quite a lot of people!

Also if you don't mind I have a couple of repo suggestions:
 - if you enable discussions, people will be able to say stuff like this there (if that's what you want obviously)
- I suggest moving the language to its own GitHub org if you're open to it: `jank-lang/jank` or something. That will help not only the visibility and discovery of the project, but might also make it feel more like a community project and might solicit more outside contributors. I'm not a psychologist, but that's my theory 👨‍⚕️

**Comment by jeaye** [comment, marked as answer]
Great suggestions. Two weeks ago, it was just me. Now there's a whole lot more community management to do. I'm still catching up. :)

1. Done!
2. Done! Turns out I already had the jank-lang org created.

**Comment by jaidetree** [comment]
> hopefully the name grows on you.

It's not so much that I don't like it, it's a fun name, I'm just thinking it's going to be a harder sell to a project manager, CTO, or tech lead in a professional team environment. I get the project is a long way from that but that's what I would love from this language since it offers:

- Great velocity with lisp + repl integration
- Plenty of learning resources since we can just refer to the 10+ years of Clojure content out there and use a good portion of code that's available
- Safety with static types, but opt-in so we have it only where it's really needed

This is exactly what I've been trying to find out of a language!

**Reply by cogwheel** [reply]
I wonder if people feel the same way about Rust. The plain English meaning almost always has negative connotations (collapsed buildings, sunken ships, etc.)

But humans are pretty adaptable. If you say the same word over and over again, it starts to lose all meaning because you're using it in a meaningless way. This happens within seconds.

As soon as people start using "jank" or "rust" to refer to a programming language, then the other meanings go out the window (at least in situations where the context is programming languages).

IMO Technical directors should know better, officers shouldn't care unless it's actively offensive, discriminatory, etc. It's their loss if they let something superficial like that sway their judgement.

**Reply by brandonstubbs** [reply]
For those wondering [Why rust is called "rust"](https://www.reddit.com/r/rust/comments/27jvdt/internet_archaeology_the_definitive_endall_source)
